<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AffichageInteraction</class>
 <widget class="QWidget" name="AffichageInteraction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>410</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>477</width>
    <height>410</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>477</width>
    <height>410</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Affichage Interaction</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>421</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Titre</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Titre_Edit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Todo</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QListWidget" name="Todo_Liste"/>
        </item>
        <item>
         <widget class="QDateEdit" name="dateEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="close_btn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>370</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>fermer</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
